notes/ltc-transfer-penalty.vbs

```diff
@@ -140,71 +137,90 @@ IF transfer_amount <> "" THEN transfer_amount = "$" & transfer_amount
+          EditBox 100, 180, 215, 15, hardship_waiver_details
+          EditBox 100, 200, 215, 15, assets_transfered
+          EditBox 100, 220, 215, 15, case_action
+          EditBox 120, 245, 80, 15, worker_signature
+          ButtonGroup ButtonPressed
+            OkButton 205, 245, 50, 15
+            CancelButton 260, 245, 50, 15
+            PushButton 10, 15, 100, 15, "EPM: Baseline Date Criteria", EPM_baseline
+            PushButton 110, 15, 80, 15, "EPM: Transfer Penalty", EPM_transfer_penalty
+            PushButton 190, 15, 135, 15, "Uncompensated Transfers TE02.14.27", TE_transfers
+          GroupBox 5, 5, 325, 30, "LTC- Transfer Penalty Resources"
+          Text 15, 50, 50, 10, "Baseline date:"
+          Text 170, 50, 80, 10, " Transfer period begins:"
+          Text 15, 70, 45, 10, "Transfer date:"
+          Text 140, 70, 110, 10, " Last full month of transfer period:"
+          Text 5, 90, 55, 10, "Transfer amount:"
+          Text 170, 90, 80, 10, " Partial penalty amount:"
+          Text 5, 120, 115, 10, "Date of application or LTC request:"
+          Text 5, 140, 115, 10, "Date client was otherwise eligible:"
+          Text 35, 160, 80, 10, "Hardship Waiver Status:"
+          Text 15, 185, 85, 10, "   Hardship waiver details:"
+          Text 5, 205, 95, 10, "Describe Assets Transferred:"
+          Text 50, 225, 50, 10, "Actions taken:"
+          Text 55, 250, 60, 10, "Worker signature:"
+        EndDialog
+           
+        Dialog Dialog1
+        cancel_confirmation 	'asks user if they really want to cancel.  If yes, then script stops.  If no, loops back to dialog.
+        
+        If ButtonPressed > 100 Then
+            err_msg = "Loop"
+
+            If ButtonPressed = EPM_baseline Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe http://hcopub.dhs.state.mn.us/epm/2_4_1_3_1.htm?rhhlterm=baseline%20date&rhsearch=baseline%20date"
+            If ButtonPressed = EPM_transfer_penalty Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe http://hcopub.dhs.state.mn.us/epm/2_4_1_3_2.htm?rhhlterm=ltc&rhsearch=LTC"
+            If ButtonPressed = TE_transfers Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://hennepin.sharepoint.com/:w:/r/sites/hs-es-poli-temp/_layouts/15/Doc.aspx?sourcedoc=%7B4980f2be-4d02-498a-a31c-77970e663c6a%7D&action=view&wdAccPdf=0&wdparaid=5A58606E"
+        Else
+            If (len(baseline_date) < 8 or IsDate(baseline_date) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid baseline date in the MM/DD/YYYY format.  See policy reference at the bottom of the dialog box if you are unsure of how to determined the baseline date."
+            If period_begins = "" then err_msg = err_msg & vbNewLine & "You must enter the start date of the transfer penalty."
+            If (transfer_date = "" or IsDate(transfer_date) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid transfer date."
```

Comment:
Lines 197 - 202 and Lines 204-207 I recommend adding trim's to the variables if they are mandatory.